jupyter/transfer_learning_on_cifar10.ipynb

```diff
@@ -4,43 +4,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Transfer Learning on CIFAR-10 Dataset\n",
+    "# Train your first model\n",
     "\n",
+    "This is the second of our [beginner tutorial series](https://github.com/awslabs/djl/tree/master/jupyter/tutorial) that will take you through creating, training, and running inference on a neural network. In this tutorial, you will learn how to train an image classification model that can recognize handwritten digits.\n",
     "\n",
-    "## Introduction\n",
+    "## Preparation\n",
     "\n",
-    "In this tutorial, you learn how to train an image classification model using [Transfer Learning](https://en.wikipedia.org/wiki/Transfer_learning). Transfer learning is a popular machine learning technique that uses a model trained on one problem and applies it to a second related problem. Compared to training from scratch or designing a model for your specific problem, transfer learning can leverage the features already learned on a similar problem and produce a more robust model in a much shorter time.\n",
-    "\n",
-    "Train your model with the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset which consists of 60,000 32x32 color images in 10 classes. As for the pre-trained model, use the ResNet50v1[1] model. It's a 50 layer deep model already trained on [ImageNet](http://www.image-net.org/), a much larger dataset consisting of over 1.2 million images in 1000 classes. Modify it to classify 10 classes from the CIFAR-10 dataset.\n",
-    "\n",
-    "![The CIFAR-10 Dataset](https://djl-ai.s3.amazonaws.com/resources/images/cifar-10.png)\n",
-    "<center>the CIFAR10 dataset</center>\n",
-    "\n",
-    "\n",
-    "## Pre-requisites\n",
-    "This tutorial assumes you have the following knowledge. Follow the READMEs and tutorials if you are not familiar with:\n",
-    "1. How to setup and run [Java Kernel in Jupyter Notebook](https://github.com/awslabs/djl/blob/master/jupyter/README.md)\n",
-    "2. Basic components of Deep Java Library, and how to [train your first model](https://github.com/awslabs/djl/blob/master/jupyter/tutorial/train_your_first_model.ipynb).\n",
-    "\n",
-    "\n",
-    "## Getting started\n",
-    "Load the Deep Java Libarary and its dependencies from Maven:"
+    "This tutorial requires the installation of the Java Jupyter Kernel. To install the kernel, see the [Jupyter README](https://github.com/awslabs/djl/blob/master/jupyter/README.md)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Would you please clear output and save the jupyter notebook:

Kernel -> Restart & clear output

Reply:
Please also squash all commits into one and force push your changes:

```
git rebase -i HEAD~8
```

Reply:
Done